src/accounts/AccountPage.tsx

```diff
@@ -0,0 +1,67 @@
+// Libraries
+import React, {FC, useContext, useState} from 'react'
+import {Button, IconFont, Overlay, Page} from '@influxdata/clockface'
+
+// Utils
+import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
+import UserAccountProvider from './context/userAccount'
+import AccountTabContainer from './AccountTabContainer'
+
+import {UserAccountContext} from 'src/accounts/context/userAccount'
+
+import {SwitchAccountOverlay} from 'src/accounts/SwitchAccountOverlay'
+
+const AccountAboutPage: FC = () => {
+  const {userAccounts} = useContext(UserAccountContext)
```

Comment:
what's the reasoning for using context here rather than a reducer?

Reply:
i was following the pattern of the other items /pages that use quartz (billing, operator ro).  I use it in multiple places.